api/incidents/views.py

```diff
@@ -15,6 +16,22 @@
 from django.http.response import Http404
 
 
+# class StandardResultsSetPagination(PageNumberPagination):
```

Comment:
Remove this commented code before pushing to GitHub

Reply:
Already removed.